src/wrapper/data_chunk.ml

```diff
@@ -29,6 +29,26 @@ let get_opt t type_ idx =
 
 let free t ~here = Resource.free t.data_chunk ~here
 
+let to_string_hum ?(bars = `Unicode) ~column_count t =
+  match t.length, column_count with
+  | 0, _ -> "" (* Empty data chunk *)
+  | _, 0 -> "" (* No columns to display *)
+  | _, _ ->
+    (* Convert `None to `Ascii since to_string_noattr doesn't accept `None *)
```

Comment:
Why do we accept `None when it's not supported? We should mimic the API for `Result_.to_string_hum`.